mage_ai/io/airtable.py

```diff
@@ -0,0 +1,100 @@
+import logging
+
+from pandas import DataFrame
+from pyairtable.api import Api
```

Comment:
you'll need to add pyairtable to requirements.txt and setup.py

Reply:
ok will do this